### DISCUSSION TITLE
Sorting functionality Swell backend not working

### DISCUSSION THREAD

**Discussion by Strongeyed** [swellstores/community#294] [Help] [answered]
The sorting functionality for categories is currently not working, changes are not reflected when made.

<img width="802" alt="Screenshot 2023-07-03 at 10 25 59 AM" src="https://github.com/swellstores/community/assets/128098520/570bba6b-e897-4a99-8c19-29c3b96608b9">

**Comment by logeshswell** [comment, marked as answer]
If you mean that the changes are not reflected in the admin dashboard, then it would reflect only when you save the changes by clicking **Save category** button.
